xopt/pydantic.py

```diff
@@ -33,9 +22,18 @@ class Config:
     type: lambda x: f"{x.__module__}.{x.__name__}",
     # for encoding instances of the ObjType}
     ObjType: lambda x: f"{x.__module__}.{x.__class__.__qualname__}",
+    np.ndarray: lambda x: x.tolist(),
```

Comment:
We should probably try to generalize this, like in https://github.com/slaclab/lume-base/blob/8c548e11672abce3a0cfc22b970b343d46ddba42/lume/tools.py#L8
It may not be possible with the way pydantic works though.